qr-code-auth/README.md

```diff
@@ -0,0 +1,277 @@
+|                       [GET /auth/qr-code-auth](#get-authqr-code-auth)                       |           Retrieves all the authentication document belonging to a specified user            |
+| [PATCH /auth/qr-code-auth](#patch-authqr-code-authauthorization_statusauthorization_status) | Updates the authorization_status field of an existing qr-code-auth document for the specified user. |
+
+## POST /auth/qr-code-auth
+
+    Creates a new qr-code-auth document.
+
+- **Params**  
+  None
+- **Query**
+  None
+- **Body**
+  - Attributes
+    - **user_id** (required, string): Specifies the user id for the document.
+    - **device_info** (required, string): Specifies the device information associated with the authentication.
+- **Headers**
+  - Content-Type: application/json
+- **Cookie**
+  None
+- **Success Response:**
+  - **Code:** 201
+    - **Content:**
+      ```json
+      {
+        "message": "String",
+        "data": {
+          "user_id": "String",
+          "device_info": "String",
+          "authorization_status": "String",
+          "access_token": "String"
+        }
+      }
+      ```
+- **Error Response:**
+
+  - **Code:** 400
+
+    - **Content:**
+      ```json
+      {
+        "message": "Bad Request"
+      }
+      ```
+
+  - **Code:** 409
+
+    - **Content:**
+      ```json
+      {
+        "message": "The authentication document has already been created"
+      }
+      ```
+
+  - **Code:** 500
+    - **Content:**
+      ```json
+      {
+        "message": "The server has encountered an unexpected error. Please contact the administrator for more information."
+      }
+      ```
+
+- **Example for user authentication document creation request:**
+  POST /auth/qr-code-auth<br/>
+  Content-Type: application/json<br/>
+  Request-Body:<br/>
+
+  ```json
+  {
+    "device_info": "t5k77PHnuDSrgEzvMJAj",
+    "user_id": "NLFSj7Kz30oHgolfIZtJ"
+  }
+  ```
+
+  Response :
+  Status 201<br/>
+  Content-Type: application/json<br/>
+
+  ```json
+  {
+    "data": {
+      "user_id": "SooJK37gzjIZfFNH0tlL",
+      "device_info": "t5k77PHnuDSrgEzvMJAj",
+      "authorization_status": "NOT_INIT",
+      "access_token": ""
+    },
+    "message": "authentication document created successfully."
+  }
+  ```
+
+## GET /auth/qr-code-auth
+
+Retrieves THE authentication document.
+
+- **Params**
+  None
+- **Query**
+
+  - user_id : Specifies the ID of the User whose authentication document will be retrieved
+
+- **Body**  
+  None
+- **Headers**  
+  None
+- **Cookie**  
+  None
+- **Success Response:**
+
+  - **Code:** 200
+
+    - **Content:**
+
+    ```json
+    {
+      "message": "String",
+      "data": {
+        "user_id": "String",
+        "device_info": "String",
+        "authorization_status": "String",
+        "access_token": "String"
+      }
+    }
+    ```
+
+- **Error Response:**
```

Comment:
GET api returns data which has access token. And we can pass any users Id and get their access tokens right?
Then they will be able to login on behalf of the actual user?
Will it not be concerning?

Correct me if I am missing something please

Reply:
@shreya-mishra @sakshambhatt shall we remove access token from GET Response ?

Reply:
Agreed! sure we can do that

Reply:
since we are not going to store the `acesstoken`, i don't think we will be able to return it, on a get call.